doc/modules/linear_model.rst

```diff
@@ -839,13 +853,13 @@ with 'log' loss.
    :ref:`l1_feature_selection`.
 
 :class:`LogisticRegressionCV` implements Logistic Regression with
-builtin cross-validation to find out the optimal C parameter.
+builtin cross-validation to find out the optimal C and l1_ratio parameters.
 "newton-cg", "sag", "saga" and "lbfgs" solvers are found to be faster
 for high-dimensional dense data, due to warm-starting. For the
-multiclass case, if `multi_class` option is set to "ovr", an optimal C
-is obtained for each class and if the `multi_class` option is set to
-"multinomial", an optimal C is obtained by minimizing the cross-entropy
-loss.
+multiclass case, if `multi_class` option is set to "ovr", an optimal C and
+l1_ratio is obtained for each class and if the `multi_class` option is set to
```

Comment:
Wait, we're not using accuracy here or scoring? That seems odd. Is that wording really correct?

Reply:
@NicolasHug can you please fix this, even though it wasn't your error?